OTA 620026 Improve acrolinx score for classic sign out guide (#4435)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-users-out/main/springboot/addbaseuri.md

> Spring Boot supports sign-out handling right out of the box. Most of the defaults for the security components can be easily configured. See [Spring Security Logout Handling](https://docs.spring.io/spring-security/site/docs/current/reference/html5/#jc-logout).

Reviewer: `out of the box` seems a little colloquial. What if we adjust it While we're here?

```suggestion
Spring Boot supports sign-out handling by default. Most of the security components default settings can be easily configured. See [Spring Security Logout Handling](https://docs.spring.io/spring-security/site/docs/current/reference/html5/#jc-logout).
```

Author: Seems good - fixed
